app/controllers/sessions_controller.rb

```diff
@@ -33,4 +25,12 @@ def destroy
     flash[:notice] = "You have successfully logged out!"
     redirect_to "/"
   end
-end
+
+  private
+
+  def login_redirect
+    redirect_to admin_path if current_admin?
```

Comment:
Great work on refactoring the redirect paths

Reply:
I agree great refactor!